challenges/qr_code_component/index.html

```diff
@@ -30,33 +30,54 @@
   </head>
```

Comment:
Favicon is missing, so either include it in the header or remove it from `images` directory.

Reply:
Default favicon was left, removed to challenge one